pom.xml

```diff
@@ -148,14 +148,23 @@
             <optional>true</optional>
         </dependency>
 
-        <!-- For simple Json Serialization -->
+        <!-- Apache Curator Recipes - Optional Dependency -->
+        <!-- Only required if you make use of the ZookeeperWatchTrigger -->
         <dependency>
-            <groupId>com.googlecode.json-simple</groupId>
-            <artifactId>json-simple</artifactId>
```

Comment:
Swapping this to Gson because Gson lets us deserialize directly to a class, where this does not.